From cc3a4cb14169666e71f674c656a8642712d70a1c Mon Sep 17 00:00:00 2001
From: Johnny Mnemonic <jm@machine-hall.org>
Date: Thu, 5 Jun 2025 18:34:05 +0200
Subject: [PATCH 29/33] Fix build after
 c4be334400508e6aa007f99ccc36c414f793c387

by dropping sysdeps/ia64/fpu/math_err.c like it's done for i386 and m68k.
---
 sysdeps/ia64/fpu/math_err.c | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 sysdeps/ia64/fpu/math_err.c

diff --git a/sysdeps/ia64/fpu/math_err.c b/sysdeps/ia64/fpu/math_err.c
deleted file mode 100644
index 1cc8931700..0000000000
--- a/sysdeps/ia64/fpu/math_err.c
+++ /dev/null
@@ -1 +0,0 @@
-/* Not needed.  */
-- 
2.25.1

